controller.go

```diff
@@ -253,6 +253,7 @@ func (c *controller) NewNetwork(networkType, name string, options ...NetworkOpti
 		id:          types.UUID(stringid.GenerateRandomID()),
 		ctrlr:       c,
 		endpoints:   endpointTable{},
+		dbExists:    false,
```

Comment:
You don't need to explicitly set this to false. The variable is created with the zero value of the field which for `bool` is `false`